### ISSUE TITLE
Trying to Change Label Font of Node in a Graph

### ISSUE THREAD

**Issue by saurabhheda** [bqplot/bqplot#1138]
I wanted to change the Font and color of Label in a node of graph but not sure how can I achieve this. 

 My label looks like below where text is messy also there is an extra shadow which is making my node difficult to read. 

![image](https://user-images.githubusercontent.com/2179996/82119539-123f0100-979d-11ea-86e3-b5ffde2916c0.png)

I tried something like this but doesn't work. Anyway by which we can change the color and font style and font of a node. 

graph_prod.label_font_style = {'fill': 'pink', 'font-size': '20px'}

**Comment by martinRenou** [comment]
Indeed it does not seem possible to change the graph font. The `label_font_style` is a `Bar` property so you won't be able to use it for the `Graph`.

**Comment by saurabhheda** [comment]
@martinRenou  Yes you are right I was just trying and then later on figured out that this is the property of Graph. Is there any workaround for this?

**Comment by martinRenou** [comment]
I am not sure. You could still try to inject your own CSS in the page, but that's definitely not the ideal solution.

It should be added to the Graph API.

**Comment by saurabhheda** [comment]
How can I inject CSS any documentation or piece of code you want me to go through for this

**Comment by martinRenou** [comment]
You can look at the answer here for injecting custom CSS: https://stackoverflow.com/a/33570738/9734668
